### PR TITLE
Setup clickhouse infrastructure

### LINKED ISSUE #137: Setup Clickhouse with Airflow
Add Clickhouse capabilities to Airflow repo.

* Add clickhouse do Docker testing stack
* Add clickouse credentials to .env
* Add clickhouse migrations and db helpers as library

### PR DESCRIPTION
**Pull request by jakobkolb**
Fix #137

WIP:
- [x] Setup clickhouse in airflow docker stack for testing 
- [x] Add clickhouse helpers and migration management in library
- [x] Add clickhouse example dag
- [x] Run migrations during CD
- [ ] Add clickhouse production secrets

### CODE REVIEW

**Review by davhin — CHANGES_REQUESTED** [review]
I like it! Some minor things

- please remove the email address from the DAG arguments, as those emails will not be spent
- we still need a CLI around the clickhouse migration tool for CI/CD

File: services/airflow/src/dags/csv_download_to_clickhouse/csv_download_to_clickhouse.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    context = create_db_context()
+    insert_dataframe(context, table, data.set_index("col1"))
+    teardown_db_context(context)
+    return context["credentials"]
+
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "email": ["jakob.j.kolb@gmail.com"],
```
[the remark this answers is not in the document]
Comment (jakobkolb) [reply services/airflow/src/dags/csv_download_to_clickhouse/csv_download_to_clickhouse.py]: Right. Cleaned that up also in all other places.

**Comment by jakobkolb** [comment]
Blocked by [this](https://github.com/rocs-org/legendary-octo-spork/issues/9)

**Comment by davhin** [comment]
> @davhin Blocked by [this](https://github.com/rocs-org/legendary-octo-spork/issues/9)

fixing rn

File: .gitsecret/paths/mapping.cfg [hunk 2]
```diff
@@ -1,3 +1,4 @@
 .github/workflows/hu-berlin.ovpn:105dd46ab38f595a1a336d2b8d26f1ca40cf72e74f60b93530a4ccb8a22a6595
 services/airflow/.env:d21b85765cb97c4790bd1927bbdc4a00b17782df565ca12605270773ac983ca3
-services/airflow/.env.prod:7ec9839812b6346213a0060da85fb73493cb211e0ab26a37ce22577e563056f9
```
Comment (davhin) [inline .gitsecret/paths/mapping.cfg:3]: This is probably part of the problem @jakobkolb

**Comment by jakobkolb** [comment]
Superseded by #149

**Pull request closed without merging**